libs/cuda_support/include/hpx/cuda_support/cuda_future_helper.hpp

```diff
@@ -51,7 +50,7 @@
 //        auto fut = helper.async(&call_trivial_kernel, 3.1415);
 //
 // -------------------------------------------------------------------------
-namespace hpx { namespace compute { namespace util
+namespace hpx { namespace cuda
```

Comment:
Please add definitions to the forwarding headers that ensure backwards compatibility of the symbols defined in here.